lib/sycamore/sycamore/utils/aryn_config.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import yaml
+import logging
+
+_DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+
+
+class ArynConfig:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+
+def get_aryn_api_key(requested_yaml_config_path: str = "") -> str:
```

Comment:
`requested_yaml_config_path` is a very long variable name; how about `config_path`?

Reply:
Changed.